<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QDialog" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogIn</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="newUser_groupBox">
     <property name="title">
      <string>Регистрация</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="newUser_verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="newLogin_horizontalLayout">
          <item>
           <widget class="QLabel" name="newLogin_label">
            <property name="text">
             <string>Логин</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="newLogin_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="newPswd_horizontalLayout">
          <item>
           <widget class="QLabel" name="newPswd_label">
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="newPswd_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="newPswd2_horizontalLayout">
          <item>
           <widget class="QLabel" name="newPswd2_label">
            <property name="text">
             <string>Повторите пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="newPswd2_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="saveUser_pushButton">
          <property name="text">
           <string>Готово</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_pushButton">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Login_groupBox">
     <property name="title">
      <string>Войти</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="userdata_verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="login_horizontalLayout">
          <item>
           <widget class="QLabel" name="login_label">
            <property name="text">
             <string>Логин</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="login_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pswd_horizontalLayout">
          <item>
           <widget class="QLabel" name="pswd_label">
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pswd_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="next_pushButton">
          <property name="text">
           <string>Войти</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="new_pushButton">
          <property name="text">
           <string>Регистрация</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
